src/index.ts

```diff
@@ -50,13 +50,15 @@ const renderWithCarbon = ({
   rect,
   layout,
   changeSortOrder,
+  flags,
 }: RenderWithCarbonArguments) => {
   if (env.carbon && changeSortOrder && theme) {
-    render({ rootElement, layout, model, manageData, theme, selectionsAPI, changeSortOrder, app, rect });
+    render({ rootElement, layout, model, manageData, theme, selectionsAPI, changeSortOrder, app, rect, flags });
   }
 };
 
 export default function supernova(env: Galaxy) {
+  const { flags } = env;
```

Comment:
We only need isEnabled, so you can destruct that and only pass that function along

Reply:
I thought it is more readable if I pass it as `flags` than `isEnable` but can change it.

Reply:
you can just rename it to `isFlagEnabled` if you want